lib/berkshelf/file_syncer.rb

```diff
@@ -58,6 +58,11 @@ def sync(source, destination, options = {})
         [exclude, "#{exclude}/*"]
       end.flatten
 
+      # let glob expand the source directory in case it is an abbreviated windows
+      # user directory: C:/Users/MATTWR~1/AppData/Local/Temp
+      # so that it matches the parent of source_files
+      source = glob(source).first
```

Comment:
I think `file.expand_path` would be a simpler way of expressing this, and doesn't run the risk of treating glob characters specially. As in https://github.com/berkshelf/berkshelf/pull/1608